src/set/entities/set.entity.ts

```diff
@@ -4,20 +4,20 @@ import { Status } from "../../shared/enums/status.enum";
 import { Language } from "../../shared/enums/language.enum";
 import { Task, TaskSchema } from "./task.entity";
 
-@Schema({ timestamps: true })
+@Schema({ timestamps: true, _id: true })
 export class Set {
+
+    _id: ObjectId
+    
     @Prop({ required: true, index: true })
     name: string
 
-    @Prop({ required: false, type: [{ type: TaskSchema}] })
+    @Prop({ required: false, type: [{ type: TaskSchema }] })
     tasks: Task[]
 
     @Prop({ default: Status.ACTIVE })
     status: Status | Status.ACTIVE
 
-    @Prop({ default: "", index: true })
```

Comment:
we removed descriptions? (not a code issue just not entirely sure if this is an oversight or a design choice)

Reply:
yes we removed descriptions for the sake of simplicity, the name of a Set should be catchy enough.